### PR TITLE
Add first sketch for new status page

### PR DESCRIPTION
**Pull request by walac**
This is a WIP for the new status page. Currently, it polls only a few services.

### CODE REVIEW

**Comment by djmitche** [comment]
Looks good so far :)

File: build-files.js [hunk 1]
```diff
@@ -98,6 +98,12 @@ module.exports = [
   'one-click-loaner/connect/index.jade',
   'one-click-loaner/connect/app.jsx',
   'one-click-loaner/connect/connect.less',
+
+  // Status
+  'status/index.jade',
+  'status/status.jsx',
```
Comment (jonasfj) [inline build-files.js]: isn't status included in app.jsx, then there is no need for this...

File: menu.js [hunk 2]
```diff
@@ -207,5 +207,15 @@ module.exports = [
     description: [
       "Manage credentials stored in the browser."
     ].join("\n")
+  },
+  {
+    title:        "Taskcluster status",
```
Comment (jonasfj) [inline menu.js]: Can we just call "Status" ?

File: menu.js [hunk 3]
```diff
@@ -207,5 +207,15 @@ module.exports = [
...
+    link:         "/status/",
+    icon:         "cogs",
+    display:      "true",
```
Comment (jonasfj) [inline menu.js]: This should be a boolean

File: status/status.jsx [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+];
+
+export let StatusChecker = React.createClass({
+  propTypes: {
+    up: React.PropTypes.bool.isRequired
+  },
+  render: function() {
+    let image = this.props.up ? "green-check.png" : "red-check.png";
+    return (
+      <img className="img-check img-fluid center-block" src={image}/>
```
Comment (jonasfj) [inline status/status.jsx]: If you get bored of png, play with awesome font:
```
<span className="fa-stack">
              <i className="fa fa-square fa-stack-2x fa-fw"></i>
              <i className="fa fa-terminal fa-stack-1x fa-fw fa-inverse"></i>
</span>
```

This is what I did in loaner to make the big icon fonts... I also had some css to change font-size :)


Anyways, that's not blocking this... Just something we can play around with some day...
As far as I'm aware if there is no XSS issues in the `/status` folder you are now the owner and designer of that tool :)

File: status/status.jsx [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+            return <Service name={service.name} key={service.name} poll={service.poll}/>;
+          })}
+        </bs.ButtonToolbar>
+      </div>
+    );
+  }
+});
+
+export let TaskclusterStatus = React.createClass({
+  render: function() {
```
Comment (jonasfj) [inline status/status.jsx]: note... You can do: `render() {` in es6

**Comment by jonasfj** [comment]
@walac fix issues in `build-files.js` and `menu.js` then merge...

r+

File: status/status.jsx (hunk 4 above)
Comment (walac) [inline status/status.jsx]: drevil

**Pull request merged**